.github/workflows/auto-update.yml

```diff
@@ -35,10 +35,53 @@ jobs:
           echo "The latest version of Tor on the website is:  ${{ steps.latest_version.outputs.latest_version_formatted }} (${{ steps.latest_version.outputs.latest_version }})"
           echo "The latest version of Tor in the manifest is: ${{ steps.manifest_version.outputs.manifest_version }}"
 
+  update-build-scripts:
+    name: Update Build Scripts
+    runs-on: ubuntu-latest
+    needs: ['check']
+    outputs:
+      pull_request_url: ${{ steps.create_pull_request.outputs.pull-request-url }}
+    if: ${{ needs.check.outputs.latest_version_formatted != needs.check.outputs.manifest_version }}
+    steps:
+      - name: Checkout
+        uses: actions/checkout@v2
+        with:
+          repository: tor-actions/tor-build-scripts
+          ref: master
+          token: ${{ secrets.TOR_BUILD_SCRIPTS_TOKEN }}
```

Comment:
Can `GITHUB_TOKEN` be used here? If not, I guess `TOC_ACTION_ACCESS_REPO` should be used instead.

Reply:
Yes, I've updated that now. 👍🏻 I've also updated the other reference to use `TOC_ACTION_ACCESS_REPO`. Hopefully that's all good.

Reply:
LGTM